yt_dlp/extractor/telemundo.py

```diff
@@ -0,0 +1,58 @@
+# coding=utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    try_get,
+    HEADRequest,
+    unified_timestamp
+)
+
+
+class TelemundoIE(InfoExtractor):
+
+    _VALID_URL = r'https?:\/\/(?:www\.)?telemundo\.com\/.+?(?=video)video\/[^\/]+(?P<id>tmvo\d{7})'
```

Comment:
why `(?=video)`?

Reply:
```suggestion
    _VALID_URL = r'https?:\/\/(?:www\.)?telemundo\.com\/.+?video\/[^\/]+(?P<id>tmvo\d{7})'
```

Hey, you never replied...